lib/src/presentation/widget/show_card_properties.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+
+import '../../core/util/constants.dart';
+import 'card_text.dart';
+
+Widget showPropertyIfExists(String? cardParam, [String? text]) {
+  if (cardParam != null) {
+    return CardText(
+      text: text ?? cardParam,
+      isTitle: text != null ? Constants.titleIsTrue : Constants.titleIsFalse,
+    );
+  } else {
+    return const SizedBox.shrink();
+  }
+}
```

Comment:
missed the extra line at the end of file, add it please

Reply:
Done